src/seedu/addressbook/AddressBook.java

```diff
@@ -146,6 +151,17 @@
     private static final int PERSON_DATA_COUNT = 3;
 
     /**
+     * Case insensitive person comparator.
+     */
+    private static final Comparator<String[]> PERSON_COMPARATOR = new Comparator<String[]>() {
+        public int compare(String[] person, String[] otherPerson) {
```

Comment:
Good choice of using comparator. Just a good practice to follow: you may want to include the "override" keyword here as you are overwrite the default underlying compare method.